collector.go

```diff
@@ -101,6 +137,38 @@ func NewExporter(connectionString string, namespace string, logger log.Logger) *
 	}
 }
 
+// Query SHOW LISTS, which has a series of rows, not columns.
+func queryShowLists(ch chan<- prometheus.Metric, db *sql.DB, logger log.Logger) error {
+	rows, err := db.Query("SHOW LISTS;")
+	if err != nil {
+		return errors.New(fmt.Sprintln("Error running SHOW LISTS on database: ", err))
```

Comment:
Errors should start lowercase, as they may be prefixed by more errors.

Reply:
Fixed these. Thanks.